samples/v2/precompiled/PhoneVerification.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+#if NETSTANDARD2_0
```

Comment:
Since we now only target .NET Standard 2.0, we can remove all this conditional compilation.